client/color/color_widows.go

```diff
@@ -0,0 +1,63 @@
+//go:build windows
```

Comment:
I think you can remove this line. The Go compiler recognizes when the file name ends with `_windows.go` and only includes it in the build if the OS is Windows.

(You also have a typo in the filename -- should be `color_windows.go`)